command/untaint.go

```diff
@@ -0,0 +1,184 @@
+package command
+
+import (
+	"fmt"
+	"log"
+	"strings"
+)
+
+// UntaintCommand is a cli.Command implementation that manually untaints
+// a resource, marking it as primary and ready for service.
+type UntaintCommand struct {
```

Comment:
We'll also need to add this into `commands.go`, otherwise `terraform untaint` won't work.